### PR TITLE
Create a file for humans too

### PR DESCRIPTION
**Pull request by nelsonmestevao**
We already have a file for robots. What about one for us? That's what
this pull request tries to fix.

Read more about this [here](http://humanstxt.org/).

### CODE REVIEW

**Comment by ZePedroResende** [comment]
The logo looks kinda stretched 😢

File: public/humans.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (ZePedroResende) [inline public/humans.txt]: stretched

File: public/humans.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                                  `.
+                               `-/++/:-`
```
Comment (ZePedroResende) [inline public/humans.txt]: The logo looks kinda stretched 😢

Comment (nelsonmestevao) [reply public/humans.txt]: I made an update.

**Pull request merged**